v2/terraform_flannel.py

```diff
@@ -14,6 +14,7 @@
 p.add("--ansibleBranch", default="master", help="Ansible Repository branch for ovn-ovs playbooks.")
 p.add("--flannelMode", default="overlay", help="Option: overlay or host-gw")
 p.add("--containerRuntime", default="docker", help="Container runtime to set in ansible: docker / containerd.")
+p.add("--retries-ansible-adhoc", type=int, default=5, help="Number of retries Ansible adhoc command should do.")
```

Comment:
Should not be named retries-ansible-adhoc. remoteCmdRetries or something like that would be better. _copyTo, _copyFrom and _runRemoteCmd are interfaces so it must be indicated that they have retry functionality regardless of the specific implementation per deployer.